### PR TITLE
Notfound extension

### PR DESCRIPTION
**Pull request by daschuer**
This aims to fix https://github.com/mixxxdj/manual/issues/395
This tragets exactly our issue
https://sphinx-notfound-page.readthedocs.io/en/latest/

### CODE REVIEW

**Comment by daschuer** [comment]
The deploy preview 404 page is no completely broken. This is expected, because it does not have the folder /en/ 
We need to actually deploy this to find out if it is working.

File: build_html.sh [hunk 1]
```diff
@@ -26,6 +26,7 @@ do
         printf '/:version/%s/* /:version/%s/404.html  404\n' "$lang" "$lang" >> build/html/_redirects
         printf '/:version/*    /:version/%s/:splat    301 Language=%s\n' "$lang" "$lang" >> build/html/_redirects
     fi
+    printf '/* /latest/en/404.html  404\n' >> build/html/_redirects
```
Comment (Swiftb0y) [inline build_html.sh:29]: are you sure this redirect is needed?

Comment (daschuer) [reply build_html.sh:29]: Yes, for unknown versions, check out https://manual.mixxx.org/2.7/
I hope it works, I cannot test.

Comment (Swiftb0y) [reply build_html.sh:29]: Makes sense. Can add a comment above?

Comment (Swiftb0y) [reply build_html.sh:30]:
```suggestion
    printf '/* /latest/404.html  404\n' >> build/html/_redirects
```
should work just as well.

**Comment by daschuer** [comment]
Done

**Review by Swiftb0y — APPROVED** [review]
Thanks. LGTM

**Pull request merged**